.gitpod.yml

```diff
@@ -0,0 +1,27 @@
+image:
+    file: .gitpod.dockerfile
+ports:
+    - port: 3000
+      onOpen: open-preview
+tasks:
+    - init: >
+        bundle install --without production &&
+        cp config/database.yml.gitpod config/database.yml &&
+        mysql -e "SET @@global.sql_mode=(SELECT REPLACE(@@global.sql_mode, 'ONLY_FULL_GROUP_BY', ''));"; &&
```

Comment:
hmm there must be a syntax error in here, but I'm not sure where!

Reply:
(Also depending on the mysql version, the syntax may be different, but I believe this one is good for us as I tested it on the console. Ref: https://stackoverflow.com/questions/23921117/disable-only-full-group-by)

Reply:
@keshavsethi I kind of got stuck here but maybe you can figure out the next step? Or if @icarito has an idea?